src/app/shared/data-input/number/number-data-input.component.html

```diff
@@ -0,0 +1,8 @@
+<input #numberInput
+       [readonly]="readonly"
+       [title]="value"
+       [value]="value"
+       (blur)="onBlur()"
+       (input)="onInput($event)"
+       type="number"
+       class="border-0 w-100 h-100 px-1 bg-transparent">
```

Comment:
Shouldn't we prevent the user from writing numbers? For example: `pattern="[0-9,.]+"`

Reply:
It is done automatically by browser if you set `type="number"`. Moreover, it also shows you the right keyboard on mobile devices.